Code/V2/src/App.css

```diff
@@ -1,5 +1,6 @@
 .App {
   text-align: center;
+  background: #52A0B4;
```

Comment:
Here I changed the background color of the default Create React App screen, in order to help demonstrate what our keys will look like later on. This was not technically a necessary change.